### ISSUE TITLE
[Bug?] PlayerDeathEvent Error

### ISSUE THREAD

**Issue by Yurinann** [Badbird5907/AntiCombatLog#10]
```
[15:49:35 ERROR]: Could not pass event PlayerDeathEvent to AntiCombatLog v2.2.5
java.lang.NullPointerException: Cannot invoke "Object.toString()" because "s" is null
        at net.badbird5907.blib.utils.StringUtils.replacePlaceholders(StringUtils.java:16) ~[AntiCombatLog-2.2.5.jar:?]
        at net.badbird5907.anticombatlog.utils.StringUtils.format(StringUtils.java:17) ~[AntiCombatLog-2.2.5.jar:?]
        at net.badbird5907.anticombatlog.listener.CombatListener.onDeath(CombatListener.java:100) ~[AntiCombatLog-2.2.5.jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor357.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:75) ~[pufferfish-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[pufferfish-api-1.18.1-R0.1-SNAPSHOT.jar:git-Pufferfish-46]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[pufferfish-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:630) ~[pufferfish-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.callPlayerDeathEvent(CraftEventFactory.java:896) ~[pufferfish-1.18.1.jar:git-Pufferfish-46]
        at net.minecraft.server.level.ServerPlayer.die(ServerPlayer.java:862) ~[?:?]
        at org.bukkit.craftbukkit.v1_18_R1.entity.CraftLivingEntity.setHealth(CraftLivingEntity.java:122) ~[pufferfish-1.18.1.jar:git-Pufferfish-46]
        at net.badbird5907.anticombatlog.AntiCombatLog.join(AntiCombatLog.java:141) ~[AntiCombatLog-2.2.5.jar:?]
        at net.badbird5907.anticombatlog.listener.ConnectionListener.onConnect(ConnectionListener.java:25) ~[AntiCombatLog-2.2.5.jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor360.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:75) ~[pufferfish-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[pufferfish-api-1.18.1-R0.1-SNAPSHOT.jar:git-Pufferfish-46]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[pufferfish-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:630) ~[pufferfish-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        at net.minecraft.server.players.PlayerList.postChunkLoadJoin(PlayerList.java:363) ~[pufferfish-1.18.1.jar:git-Pufferfish-46]
        at net.minecraft.server.players.PlayerList.lambda$placeNewPlayer$1(PlayerList.java:305) ~[pufferfish-1.18.1.jar:git-Pufferfish-46]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.tick(ServerGamePacketListenerImpl.java:308) ~[?:?]
        at net.minecraft.network.Connection.tick(Connection.java:555) ~[?:?]
        at net.minecraft.server.network.ServerConnectionListener.tick(ServerConnectionListener.java:201) ~[?:?]
        at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1642) ~[pufferfish-1.18.1.jar:git-Pufferfish-46]
        at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:483) ~[pufferfish-1.18.1.jar:git-Pufferfish-46]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1476) ~[pufferfish-1.18.1.jar:git-Pufferfish-46]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1266) ~[pufferfish-1.18.1.jar:git-Pufferfish-46]
        at net.minecraft.server.MinecraftServer.lambda$spin$1(MinecraftServer.java:319) ~[pufferfish-1.18.1.jar:git-Pufferfish-46]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

Environment: same to #9

**Comment by Badbird5907** [comment]
I know about this, and will be fixed soon. This error doesn't break anything

**Comment by Badbird5907** [comment]
fixed in 9f16443e6a651fa96006b07b93abb07b9dd2055c
